control/content/assets/css/bf_base.css

```diff
@@ -0,0 +1,882 @@
+/* General */
+@font-face {
+  font-family: 'Conv_apercu_bold';
```

Comment:
Why do we keep adding custom fonts? That should be inherited from buildfire right? Is there any specific reason?